dynamic_instrumentation/src/point/build.rs

```diff
@@ -140,6 +151,15 @@ impl<'tcx> InstrumentationBuilder<'_, 'tcx> {
         self
     }
 
+    /// Sets the priority of this instrumentation. Lower value
+    /// signifies higher priority and implies that an instrumentation
+    /// with higher priority will be placed before one with lower
+    /// priority.
+    pub fn instrumentation_priority(mut self, p: usize) -> Self {
+        self.instrumentation_priority = p;
+        self
```

Comment:
It's a bit more obvious to write self-with-a-modification using record update syntax:
```suggestion
        Self { instrumentation_priority: p, ..self }
```

Reply:
True, but it gets more complicated with something like https://github.com/immunant/c2rust/blob/a35136ec9d962e64f57980c2ef5ea3bbee0f40bc/dynamic_instrumentation/src/point/build.rs#L149-L152

it doesn't work as well, as this keeps the same style for all of these builder methods.

Reply:
opting to keep as-is